drivers/gpu/drm/panel/panel-k6-38-0e-0b-dsc.c

```diff
@@ -18,415 +18,415 @@
 #include <drm/drm_panel.h>
 
 struct k6_38_0e_0b_dsc {
-	struct drm_panel panel;
-	struct mipi_dsi_device *dsi;
-	struct drm_dsc_config dsc;
-	struct gpio_desc *reset_gpio;
-	bool prepared;
+    struct drm_panel panel;
+    struct mipi_dsi_device* dsi;
+    struct drm_dsc_config dsc;
+    struct gpio_desc* reset_gpio;
+    bool prepared;
 };
 
-static inline
-struct k6_38_0e_0b_dsc *to_k6_38_0e_0b_dsc(struct drm_panel *panel)
```

Comment:
Follow the latest upstream driver styles (Leave it as it was in the previous version)